packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added support for multiselect group items.
```

Comment:
We should list the props here:
```
Added prop `propname` for <functionality>.
```

Reply:
Updated : https://github.com/cerner/terra-framework/pull/1790/files#diff-be3ba623085d595a5e3637c4478322d99dc1649652b44e5662563e3fd2324f4f